scripts/run_backend_tests.py

```diff
@@ -308,8 +305,14 @@ def _get_test_target_classes(path):
     return result
 
 
-def main():
+def main(argv=None):
     """Run the tests."""
+    setup.main()
+    setup_gae.main()
```

Comment:
Explicitly pass empty args

Reply:
Setup and setup_gae accepts no arguments and hence its main accepts no argv